Terran-One/cw-simulate-ui
# Fix state cache issue

## Description
Current jotai/CWSimulateApp state mgmt solution is still garbage and causes lots of issues. Fixed one of them, but there will be more. We need a new solution sooner or later.

## Test steps
1. Create a new non-Terra simulation
2. Open settings drawer submenu. Chain ID & Bech32 prefix should be correct (i.e. not Terra)
